src/ui/map.js

```diff
@@ -295,6 +295,8 @@ const defaultOptions = {
  *  * [Albers](https://en.wikipedia.org/wiki/Albers_projection) equal-area conic projection as `albers`
  *  * [Equal Earth](https://en.wikipedia.org/wiki/Equal_Earth_projection) equal-area pseudocylindrical projection as `equalEarth`
  *  * [Equirectangular](https://en.wikipedia.org/wiki/Equirectangular_projection) (Plate Carrée/WGS84) as `equirectangular`
+ *  * Globe, a 3D spherical projection, as `globe`
```

Comment:
Technically, globe is a [general perspective projection](https://en.wikipedia.org/wiki/General_Perspective_projection).

But I think that there's an intuitive sense that globe isn't a projection at all, since projections are mappings of the globe to a flat surface. I think that people will understand what what "Globe" means without further explanation

```suggestion
 *  * 3d Globe as `globe`
```